content/backend/graphql-js/1-getting-started.md

```diff
@@ -256,13 +258,13 @@ mutation {
 
 There are a few things to note:
 
-- In these examples, we always query `id` and `name` of the returned `User` objects. We could potentially omit either of them. Note however when querying an object type, it is required that you query at least one of its fields in a selection set.
-- For the fields in the selection set, it doesn't matter whether the type of the root field is _required_ or a _list_. In the examples schema above, the three root fields all have different [type modifiers](http://graphql.org/learn/schema/#lists-and-non-null) (i.e. different combinations of being a list and/or required) for the `User` type:
+- In these examples, we always query `id` and `name` of the returned `User` objects. We could potentially omit either of them. Note, however, when querying an object type, it is required that you query at least one of its fields in a selection set.
+- For the fields in the selection set, it doesn't matter whether the type of the root field is _required_ or a _list_. In the example schema above, the three root fields all have different [type modifiers](http://graphql.org/learn/schema/#lists-and-non-null) (i.e. different combinations of being a list and/or required) for the `User` type:
   - For the `users` field, the return type `[User!]!` means it returns a _list_ (which itself can not be `null`) of `User` elements. The list can also not contain elements that are `null`. So, you're always guaranteed to either receive an empty list or a list that only contains non-null `User` objects.
   - For the `user(id: ID!)` field, the return type `User` means the returned value could be `null` _or_ a `User` object.
   - For the `createUser(name: String!)` field, the return type `User!` means this operation always returns a `User` object.
 
+// TODO (robin-macpherson): question #2 for Nikolas
 As you provide this information, the `Prisma` instance will get full access to your database service and can be used to resolve incoming request later on.
```

Comment:
@nikolasburk it feels random that we mentioned Prisma suddenly here. Was this intentional?

Reply:
Huh,... yeah that seems weird! Maybe a similar issue as the other one where this might be just a leftover from a previous iteration 🤔 if this doesn't make sense in this context feel free to remove it.

Reply:
Ok will do!